frontend/src/metabase/common/components/EntityPicker/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const RECENTS_TAB_ID = "recents-tab";
```

Comment:
Previously the ids of these tabs were hardcoded in multiple files. Now we have constants for them.